src/TFR.jl

```diff
@@ -50,11 +50,11 @@ function stft(audio::SampleBuf{T, 2},
     nchannels = SampledSignals.nchannels(audio)
     noverlap = windowsize - hopsize
 
-    stft = Array(Matrix, nchannels)
+    stft = Array{Matrix}(undef, nchannels)
```

Comment:
Notice that `Array{Matrix}` is type unstable. You have to do `Array{Matrix{T}}` with `T` the appropriate numeric element type (I guess `T = eltype(audio.data)`)